src/cli/internal/main.ts

```diff
@@ -26,9 +26,20 @@ export async function main(args?: string[]) {
   SetOptions(options);
 
   const ontologyUrl = options.ontology;
-  Log(`Loading Ontology from URL: ${ontologyUrl}`);
+  const filePath = options.file;
+  let result;
+  if (ontologyUrl && ontologyUrl.startsWith('https://')) {
```

Comment:
Since `ontology` is always provided by default, and `file` is null by default, I would assume it's more straightforward to structure this differently:

```
if (filePath) {
  // loadFile
} else {
  // load(ontologyUrl)
}
```

Reply:
changed